### PR TITLE
AJAX Row + Server Call Details Styling Tweaks

### PR DESCRIPTION
**Pull request by davidkpiano**
For #128 
screen shot 2017-06-27 at 2 01 54 pm

### CODE REVIEW

**Comment by mike-kaufman** [comment]
why does the image have "Load load time" in the first vertical pane?

**Comment by mike-kaufman** [comment]
please get @philliphoff  & @legomushroom  to review.

**Review by mike-kaufman — APPROVED** [review]

**Review by legomushroom — APPROVED** [review]
LGTM

**Comment by avanderhoorn** [comment]
Icon needs to be on the left side of the "200s" not there right. And good catch, it should be "Page load time".

**Comment by philliphoff** [comment]
Should the icons for each server logs and browser logs lines be smaller, to match the reduction in font size for those lines?  I find it difficult to visually distinguish between the summary stats and the specific server/browser stats, and I think it's because the icons are the same size (and so make it look like three sibling stats with an alignment issue).  Either that, or add more visual space between them (though that might make the alignment weird across columns.

I also would prefer to see the status codes/messages aligned, as I find it difficult to read as it jags left/right due to the differing lengths of the request method.  That said, the methods may not vary too much (likely being mostly GET or mostly POST/PUT), so that may not matter too much.

Otherwise, LTGM.

**Comment by davidkpiano** [comment]
I'd prefer to see them aligned too but the goal of this (at the moment, until we have some solid mocks) is consistency with Glimpse.Client, where it jags there as well:
screen shot 2017-06-27 at 4 16 49 pm

The alignment/position/size has been fixed:
screen shot 2017-06-27 at 4 15 58 pm

**Comment by philliphoff** [comment]
> consistency with Glimpse.Client, where it jags there as well

Yeah, that bugs me too.  :-)

Should there be a dot between text in "1 200s" and "1 Create"?  I get using it to separate two metrics, but the aren't multiple metrics so I would think it better to use color to differentiate between metric and "unit" (like we do with durations).

**Comment by davidkpiano** [comment]
> Should there be a dot between text in "1 200s" and "1 Create"? 

Up to @avanderhoorn, I can go either way.

**Comment by davidkpiano** [comment]
Merge conflicts resolved, LGTY? @philliphoff @avanderhoorn

**Comment by mike-kaufman** [comment]
@philliphoff, @avanderhoorn - I'm going to merge this in in about 10 minutes or so (once the node TS fixes are in) so we can cut the build.   Please reply if there's blocking feedback for this PR. 

RE design-related questions, I'd like us to get in what's here, and then we can ask chet/Nik for feedback on what's in the build and go from there.

File: src/views/data.js [hunk 1]
```diff
@@ -63,23 +64,25 @@ function updateValue(target, summary) {
...
+        content += ' &middot; ' + summary.time;
     }
     element.innerHTML = content;
 }
 function updateListingWebServices(target, statusCodes) {
-    updateCoreListing(target, Object.assign({}, statusCodes), supportedStatusCodes, 's');
+    updateCoreListing(target, Object.assign({}, statusCodes), supportedStatusCodes, true);
 }
 function updateListingDataStore(target, operationCategories) {
     updateCoreListing(target, Object.assign({}, operationCategories), supportedOperationCategories, '');
```
Comment (philliphoff) [inline src/views/data.js]: Should `''` be `false`?

Comment (davidkpiano) [reply src/views/data.js]: Yes

**Comment by davidkpiano** [comment]
@philliphoff Fix is in!

**Comment by philliphoff** [comment]
Ok, LGTM

**Comment by avanderhoorn** [comment]
I thought the dot would look better than it does. Nik/Chet what you think
about dot vs slack vs brackets?

**Pull request merged**